_posts/2016-08-24-On-Emission.markdown

```diff
@@ -35,7 +35,7 @@ For example, the route `artsy.net/artwork/glenn-brown-suffer-well` maps to the n
 
 Just as a browser knows very little about the individual content of the pages that it's rendering, the eigen chrome exists _relatively_ independent of the view controllers that are showing. 
 
-Each view controller also knows very little about each-other, so actions that trigger a new view controller are generally done by creating a string route and passing it through the routing system. I've wrote about about pattern this in [Cocoa Architecture: Router Pattern][router_pattern]. 
+Each view controller also knows very little about each-other, so actions that trigger a new view controller are generally done by creating a string route and passing it through the routing system. I've wrote about this pattern in [Cocoa Architecture: Router Pattern][router_pattern]. 
```

Comment:
This is definitely an improvement. If I may suggest, two other things here aren't English:

- Each view controller can't know about "each-other". It's _View controllers know very little about each-other._
- I've wrote is not grammatically correct, it's _I've written_ or _I wrote_.